README.md

```diff
@@ -161,7 +161,7 @@ TRIE.INSERT inserts a string into a given trie key. It can insert as many string
        (int) 0
 
 ### TRIE.CONTAINS key value
-TRIE.CONTAINS checks if a string exists in a given trie key. If the trie key does not exist, an error will be thrown. Otherwise, different messages will be printed based on whether the string is contained within the trie. If the string is contained within the trie, the integer 1 will be returned. If the string is not contained within the trie, the integer 0 will be returned. If the string is contained within the trie as a prefix but not as a word, the integer -1 will be returned.
+TRIE.CONTAINS checks if a string exists in a given trie key. If the trie key does not exist, an error will be thrown. Otherwise, different values will be returned based on whether the string is contained within the trie. If the string is contained within the trie, the integer 1 will be returned. If the string is not contained within the trie, the integer 0 will be returned. If the string is contained within the trie as a prefix but not as a word, the integer -1 will be returned.
```

Comment:
Could you clarify that calling TRIE.INSERT on a new key name creates a new trie key?

Reply:
It's already in there